source/main/modules/window-manager/index.ts

```diff
@@ -823,7 +896,7 @@ export default class WindowManager {
    *
    * @return {string[]}                             An array containing all selected files.
    */
-  async askFile (filters: FileFilter[]|null = null, multiSel: boolean = false, win?: BrowserWindow|null): Promise<string[]> {
+  async askFile (filters: FileFilter[] | null = null, multiSel: boolean = false, win?: BrowserWindow | null): Promise<string[]> {
```

Comment:
Don't make whitespace changes to code unrelated to the pull request. I don't think the Zettlr developers will approve the whitespace changes.